src/jaxsim/high_level/model.py

```diff
@@ -1479,6 +1490,88 @@ def integrate(
             },
         )
 
+    # ==============
+    # Motor dynamics
+    # ==============
+
+    @functools.partial(oop.jax_tf.method_rw, static_argnames=["joint_names"])
+    def set_motor_inertias(
+        self, inertias: jtp.Vector, joint_names: tuple[str, ...] = None
+    ) -> None:
+        joint_names = joint_names or self.joint_names()
+
+        if inertias.size != len(joint_names):
+            raise ValueError("Wrong arguments size", inertias.size, len(joint_names))
+
+        self.physics_model._joint_motor_inertia.update(
+            dict(zip(self.physics_model._joint_motor_inertia, inertias))
+        )
+
+        logging.info("Setting attribute `motor_inertias`")
```

Comment:
do we want this log message always printed?

Reply:
With a normal installation it should not get printed as the logging level is set to by default. The log message will appear only when installing in development mode: https://github.com/ami-iit/jaxsim/blob/293161cf16f486f2a2eb13ce2e963d951b3f8820/src/jaxsim/__init__.py#L48-L51